game/views.py

```
@@ -1154,109 +1154,59 @@ def password_reset_account_selection(request):
 def confirm_delete_account(request, uidb64, token):
-
     try:
-
-        uid = force_str(
-            urlsafe_base64_decode(uidb64)
-        )
-
+        uid = force_str(urlsafe_base64_decode(uidb64))
         user = User.objects.get(pk=uid)
-
     except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)